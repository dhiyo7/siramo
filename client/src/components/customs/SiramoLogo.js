import React, { Component } from 'react'
import { 
  View,
  StyleSheet, Image, ImageBackground
} from 'react-native'

const background = require('../../assets/logo/ScreenShot.png')
const backgroundImage = require('../../assets/logo/3147-compressor.jpg')

class SiramoLogo extends Component {

  render() {
    return (
    //   <ImageBackground 
    //   source={backgroundImage}
    //   style={{width: '100%', height: '100%'}}
    // >
      <View style={styles.mainContainer}>
        <Image
          style={styles.imageContainer}
          source={background}
        />
      </View>
      // </ImageBackground>
    );
  }
}

const styles = StyleSheet.create({
  mainContainer: {
    flex: 1,
    marginLeft: 20,
    justifyContent: 'center',
    padding: 16,
    paddingLeft: 0,
    height: 40,
    marginLeft: 20
  },
  imageContainer: {
    width: 160,
    height: 40,
  },
})

export default SiramoLogo;