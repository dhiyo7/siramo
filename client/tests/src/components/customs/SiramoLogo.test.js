import React from 'react'
import {
  View,
  StyleSheet, Image
} from 'react-native'
import Enzyme, { mount, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import renderer from 'react-test-renderer'
import SiramoLogo from '../../../../src/components/customs/SiramoLogo'
import { Ionicons } from '@expo/vector-icons'

Enzyme.configure({ adapter: new Adapter()})

let wrapper

beforeEach(() => {
  wrapper = shallow(<SiramoLogo />)
})

describe('<SiramoLogo /> snapshot Testing', () => {
  it('snapshot works!', () => {
    const tree = renderer.create(<SiramoLogo />)
    expect(tree).toMatchSnapshot()
  })
})