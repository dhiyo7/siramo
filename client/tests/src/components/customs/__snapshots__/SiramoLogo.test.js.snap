// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SiramoLogo /> snapshot Testing snapshot works! 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "padding": 10,
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 40,
        "marginBottom": 4,
        "marginTop": 4,
        "width": 180,
      }
    }
  />
</View>
`;
